src/tests/ftest/soak/soak.py

```diff
@@ -308,33 +385,59 @@ def job_setup(self, test_param, pool):
                 "<<FAILED: There are only {} client nodes for this job. "
                 "Job requires {}".format(
                     len(self.hostlist_clients), self.nodesperjob))
-        if self.job_manager == "slurm":
+
+        #  get the list of cmdlines for the job
+
+        if "ior" in job:
+            # commands = [[job, command, #tasks per node], []]
+            job_cmds_tasks = self.create_ior_cmdline(
+                job, pool, self.task_list)
+        elif "dmg" in job:
+            # create dmg cmdline
+            job_cmds_tasks = self.create_dmg_cmdline(job, pool)
```

Comment:
(pylint-redefined-variable-type) Redefinition of job_cmds_tasks type from list to str